addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -405,51 +407,43 @@ private CubeCommand getCommand(SendCommand sendCommand) {
 
         // send command to MAX! Cube LAN Gateway
         HeatingThermostat device = (HeatingThermostat) getDevice(serialNumber, devices);
-
         if (device == null) {
-            logger.debug("Cannot send command to device with serial number {}, device not listed.", serialNumber);
+            logger.debug("Cannot send command to device with serial number '{}', device not listed.", serialNumber);
             return null;
         }
 
-        String rfAddress = device.getRFAddress();
-        SCommand cmd = null;
-
         // Temperature setting
         if (channelUID.getId().equals(CHANNEL_SETTEMP)) {
-            if (command instanceof DecimalType || command instanceof OnOffType) {
-                DecimalType decimalType = DEFAULT_OFF_TEMPERATURE;
+            if (command instanceof DecimalType || command instanceof QuantityType || command instanceof OnOffType) {
+                double setTemp = DEFAULT_OFF_TEMPERATURE;
                 if (command instanceof DecimalType) {
-                    decimalType = (DecimalType) command;
+                    setTemp = ((DecimalType) command).doubleValue();
```

Comment:
DecimalType commands will be automatically converted to QuantityType commands (https://github.com/eclipse/smarthome/pull/5734). 
So all logic for handling DecimalType commands on QuantityType channels in bindings will be dead code.

Reply:
Yes, I know that. But what if the user creates a basic `Number` item? Should we be backward compatible or not?

Reply:
It seems to be backwards compatible too when I create a `Number` item without a dimension for the `set_point` channel of a Nest Thermostat (Nest Binding). When I change this item via a sitemap, the handler receives a `QuantityType` command.

Reply:
Alright. I trust you and I have changed it.